### ISSUE TITLE
Single GPU WaveMLPT-dw training.

### ISSUE THREAD

**Issue by Ujjawal-K-Panchal** [ggjy/Hire-Wave-MLP.pytorch#4]
Hello,
I'm trying to run your repo with a small batch size of `2` on my modest RTX2080Su. A simple single run. No parallelism. I understand it is impractical to train the whole thing here. But I just want it to run for 1 epoch -- a small sanity check.

But trying to run the file on my machine, I receive the following 2 errors (2nd one of which is blocking):

1. File `train_wave.mlp` On `line 578:` `args` does not have `max_history`. Perhaps this is an argument generated on author's machine after the first run. I choose to remove this argument from `CheckpointSaver` function. This seems to go through.

2. File `train_wave.mlp` In function `train_epoch` On `line 668:` for some reason which I don't understand, the code passes in `target.argmax(dims = -1)` to the `accuracy()` function from `timm.utils`, instead of directly feeding it `target`. This causes an error. I instead just pass in `target` and the code works. Note, that contrastingly, in function `validate` on `line 780` the code chooses to pass `target` and not the quirky argmax version.

It would really help to get some clarity on this peculiar behavior.
